### DISCUSSION TITLE
Show how the package being used with other chains

### DISCUSSION THREAD

**Discussion by foursmalltrees** [paulmillr/micro-btc-signer#72] [General]
This library is very interesting because it is dead simple to use. I would love to see how it can be used with other utxo-based chains such as Litecoin, Dogecoin, BCH, BSV, and even Liquid/elements.

Is there a way for me to ask @paulmillr to work on this like a bounty?

**Comment by paulmillr** [comment]
Not really, focusing on other things now. The community forks are very welcome though
